README.md

```diff
@@ -498,18 +498,18 @@ grep -r -l "smt" . | xargs sed -i 's/smt/my-package/'
 
 **Incremental:** Ideal for applications where you frequently add new elements and need to update the tree efficiently.
 
-**Lean Incremental:** A more memory-efficient version of the incremental Merkle tree, suitable for use cases with memory constraints.
+**Lean Incremental:** A more memory-efficient version of the incremental Merkle tree.
 
-**Sparse:** Best suited for scenarios where you need to manage a large, mostly empty set of possible elements.
+**Sparse:** Particularly useful when you need proof of non-membership.
 
-| Type                 | Library Name     | Main Feature                           | Recommended for                          | Not Recommended for                               | Used by                                                                                                                   | Proof Gen Speed                  |
-| -------------------- | ---------------- | -------------------------------------- | ---------------------------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | -------------------------------- |
-| **Incremental**      | @zk-kit/imt      | Fastest for incremental updates        | Applications requiring frequent updates  | Scenarios with very large datasets                | [Semaphore](https://github.com/semaphore-protocol/semaphore), [Tornado Cash](https://github.com/tornadocash/tornado-core) | Fast                             |
-| **Lean incremental** | @zk-kit/lean-imt | Optimized for lightweight environments | Mobile and browser-based ZK applications | High-frequency update requirements                | [zkSync](https://github.com/matter-labs/zksync), [Loopring](https://github.com/Loopring)                                  | Moderate                         |
-| **Sparse**           | @zk-kit/smt      | Handles very large sets efficiently    | Applications with large static datasets  | Frequent updates due to higher computational cost | [Aztec](https://github.com/AztecProtocol), [Zcash](https://github.com/zcash)                                              | Slow (due to large set handling) |
+| Type                 | Library Name     | Main Feature                            | Used by                                                                                                                 |
+| -------------------- | ---------------- | --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| **Incremental**      | @zk-kit/imt      | Fastest for incremental updates.        | [Semaphore V3](https://github.com/semaphore-protocol/semaphore/tree/v3.15.2), [Worldcoin](https://github.com/worldcoin) |
```

Comment:
Should the name of the libraries be a link so that people can go directly to take a look?